flow-typed/types/ui-options.js

```diff
@@ -12,5 +12,6 @@ declare type UIOptionsObject = {
   uiComponents?: Array<KPUIComponent>,
   translations?: {[langKey: string]: Object},
   locale?: string;
-  userTheme?: UserTheme
+  userTheme?: UserTheme,
+  userSettingsMenus: any[],
```

Comment:
if userSettingsMenus always has the same structure (icon, label, options, onMenuChosen) make a type for it.

Reply:
removed- refactored